staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -39,6 +39,8 @@ import (
 	"k8s.io/apiserver/pkg/endpoints/metrics"
 	"k8s.io/apiserver/pkg/registry/rest"
 	genericfilters "k8s.io/apiserver/pkg/server/filters"
+	utilopenapi "k8s.io/apiserver/pkg/util/openapi"
```

Comment:
Do we need these long names everywhere? :-/

Reply:
I think in apiserver it is normal to use a name like this, ie
```
utilfeature "k8s.io/apiserver/pkg/util/feature"
```